release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -192,6 +192,22 @@ installation does not fail {product-title} cluster installation.
 You can xref:../openshift_images/samples-operator-alt-registry.adoc#installation-restricted-network-samples_samples-operator-alt-registry[configure alternate or mirrored registries]
 to bypass this issue if it arises during your cluster installation.
 
+[id="ocp-4-4-developer-experience"]
+=== Developer experience
+
+[id="ocp-4-4-odo"]
+==== odo enhancements
+
+`odo` has a few enhancements and improvements that focus on the user experience:
+* An `odo debug info` command is added.
```

Comment:
The bulleted list isn't rendering properly because there needs to be a blank line above it.

Reply:
is added -> is now available